app/modules/transcribe/subjects.factory.js

```diff
@@ -7,7 +7,7 @@ require('./transcribe.module.js')
     .factory('SubjectsFactory', SubjectsFactory);
 
 // @ngInject
-function SubjectsFactory($q, localStorageService, zooAPI, zooAPIProject) {
+function SubjectsFactory($q, localStorageService, zooAPI, zooAPIConfig, zooAPIProject) {
```

Comment:
It's injected via `ngInject`, it refers to [zooapi.config.js](https://github.com/zooniverse/shakespeares_world/blob/master/app/modules/zoo-api/zooapi.config.js)